pom.xml

```diff
@@ -425,6 +425,11 @@
 			<artifactId>jena-core</artifactId>
 			<version>3.6.0</version>
 		</dependency>
+		<dependency>
+			<groupId>org.reflections</groupId>
+			<artifactId>reflections</artifactId>
+			<version>0.9.12</version>
+		</dependency>

```

Comment:
Why is this dependency needed? The project is built for me without issues when I comment out the dependency :thinking:

Reply:
Sorry for the late answer.
The thing is when I don't set the version to 0.9.12, then version 0.9.11 will be used. And that version seems to have a multithreading issue. 

So in the default case in [NLIWODBasedSystem](https://github.com/dice-group/gerbil/blob/c46647eebdd0558328463d7b7234d9d64123670c/src/main/java/org/aksw/gerbil/annotator/impl/qa/NLIWODBasedSystem.java#L106) the `reflections` dependency is used to retrieve names of systems in `qa.systems`. Now, when an experiment is started with multiple systems that execute the default case (e.g. `TeBaQA`,  `QAmp`,  `Platypus`), then two of threads will throw a `java.lang.IllegalStateException: zip file closed` exception. Version 0.9.12 doesn't seem to have this issue.